custom-prom.md

```diff
@@ -3,15 +3,28 @@ Prometheus Configuration Guide
 
 ## Bring your own Prometheus
 
-When integrating Kubecost with an existing Prometheus, we recommend first installing Kubecost with a bundled Prometheus ([instructions](http://kubecost.com/install)) as a dry run before integrating with an external Prometheus deployment. You can get in touch (support@kubecost.com) or via our [Slack community](https://join.slack.com/t/kubecost/shared_invite/zt-1dz4a0bb4-InvSsHr9SQsT_D5PBle2rw) for assistance.
+When integrating Kubecost with an existing Prometheus, we _strongly_ recommend installing Kubecost with a bundled Prometheus ([instructions](http://kubecost.com/install)).
```

Comment:
The sentence doesn't make sense to me. I feel like it should be a warning/note at the top of the page that we strongly recommend using the bundled prometheus.

Like line 12 below.

Reply:
excellent point.